expr/bitwise_test.go

```diff
@@ -0,0 +1,55 @@
+package expr
+
+import (
+	"encoding/binary"
+	"reflect"
+	"testing"
+
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+func TestBitwise(t *testing.T) {
+	tests := []struct {
+		name string
+		bw   Bitwise
+	}{
+		{
+			name: "Unmarshal Bitwise IPv4 case",
+			bw: Bitwise{
+				SourceRegister: 1,
+				DestRegister:   2,
+				Len:            4,
+				Xor:            []byte{0x0, 0x0, 0x0, 0x0},
+				Mask:           []byte{0xff, 0xff, 0x0, 0x0},
```

Comment:
Is this how one matches a /16 IPv4 subnet mask? If so, add a comment please.

Reply:
done